crt_portal/cts_forms/templates/forms/complaint_view/show/index.html

```diff
@@ -78,4 +77,5 @@ <h2 class="margin-right-205 margin-top-0 margin-bottom-0 backend-blue">
 {% block page_js %}
 {{ block.super }}
 <script src="{% static 'js/show_hide_narrative_summary.js' %}"></script>
+<script src="{% static 'js/edit_contact_info.js' %}"></script>
 {%endblock%}
```

Comment:
If there was an error, I noticed some lag on my computer where I saw the form render for a second and then saw the form. I reordered the scripts locally, and that made it so I didn't really notice the lag.

```suggestion
<script src="{% static 'js/edit_contact_info.js' %}"></script>
{{ block.super }}
<script src="{% static 'js/show_hide_narrative_summary.js' %}"></script>
{%endblock%}
```

Reply:
Ah interesting, I think it's because we're hiding the form upon submission. So _before_ the page reloads the browsers hides the form and reveals the originally non-editable content. Great catch! Change incoming

Reply:
Resolved w/ 3d0b77ad6b3e31b264686efc4702516c9f160d08